examples/chip-tool/gen/att-storage.h

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
Do we need these files? I personally believe they should be in the core library.

Reply:
According to current status, seems we will still put them in seperate example directories.

Reply:
this particular file is generated specifically for the app, is different for every collection of clusters